### PR TITLE
[master] Fix clipboard issue in dialog

### LINKED ISSUE #465: Copy to Clipboard" didn't work anymore in the secret dialog
it is possible to copy the secret in the cluster detail, but not in the dialog.

Bildschirmfoto 2019-09-30 um 17 39 47

### PR DESCRIPTION
**Pull request by holgerkoser**
**What this PR does / why we need it**:
Fix clipboard issue in dialog for master branch

**Which issue(s) this PR fixes**:
Fixes #465 

**Special notes for your reviewer**:

**Release note**:

```improvement user
```

### CODE REVIEW

File: frontend/src/components/CopyBtn.vue [hunk 1]
```diff
@@ -75,17 +75,28 @@ export default {
     },
     snackbarColor () {
       return this.copyFailed ? 'error' : undefined
+    },
+    clipboardOptions () {
+      const options = {
+        text: () => this.clipboardText
+      }
+      let vm = this
```
Comment (petersutter) [inline frontend/src/components/CopyBtn.vue]: start with parent

File: frontend/tests/unit/CopyBtn.spec.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+import { expect } from 'chai'
+import Vue from 'vue'
+import Vuetify from 'vuetify'
+import { mount } from '@vue/test-utils'
+import CopyBtn from '@/components/CopyBtn.vue'
+
+Vue.use(Vuetify)
+document.body.setAttribute('data-app', true)
+global.requestAnimationFrame = cb => cb()
```
Comment (petersutter) [inline frontend/tests/unit/CopyBtn.spec.js]: link to issue in testrunner

File: frontend/tests/unit/CopyBtn.spec.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+const components = { CopyBtn }
+
+describe('CopyBtn.vue', function () {
+  it('should ensure that the clipboard container is the dialog content', function () {
+    const template = '<v-dialog ref="dialog"><copy-btn ref="btn"/></v-dialog>'
+    const wrapper = mount({ template, components })
+    const { btn, dialog } = wrapper.vm.$refs
+    expect(btn.clipboard.container).to.equal(dialog.$refs.content)
```
Comment (petersutter) [inline frontend/tests/unit/CopyBtn.spec.js]: expect $refs.content not undefined

File: frontend/src/components/CopyBtn.vue (hunk 1 above)
Comment (holgerkoser) [reply frontend/src/components/CopyBtn.vue]: ✔️

File: frontend/tests/unit/CopyBtn.spec.js (hunk 2 above)
Comment (holgerkoser) [reply frontend/tests/unit/CopyBtn.spec.js]: ✔️

File: frontend/tests/unit/CopyBtn.spec.js (hunk 3 above)
Comment (holgerkoser) [reply frontend/tests/unit/CopyBtn.spec.js]: ✔️

**Pull request merged**